File: cs/src/formulas/circularSectorAreart.cs
public static class CircularSectorAreart
{
    public static double Calculate(double r, double? t = null, double? l = null)
    {
        if (r != null)
        {
            if (t != null || l != null)
            {
                if (t != null)
                    return 0.5 * r * Math.Pow((double)t, 2);
                else
                    return 0.5 * r * (double)l;
            }
        }
        throw new ArgumentException("Invalid arguments.");
    }
}

File: py/src/formulas/circularSectorAreart.py
def circular_sector_areart(r: float, t: float = None, l: float = None) -> float:
    if r is not None:
        if t is not None or l is not None:
            if t is not None:
                return 0.5 * r * (t ** 2)
            else:
                return 0.5 * r * l
